.linux-goreleaser.yaml

```diff
@@ -0,0 +1,58 @@
+project_name: rill
```

Comment:
this file would be more prone to sync issues and require more maintenance. Not sure if this is better than earlier docker image run.
cc @begelundmuller @nishantmonu51